src/bin/spin.rs

```diff
@@ -54,6 +63,8 @@ impl SpinApp {
             Self::Bindle(cmd) => cmd.run().await,
             Self::Deploy(cmd) => cmd.run().await,
             Self::Build(cmd) => cmd.run().await,
+            Self::Trigger(TriggerCommands::Http(cmd)) => cmd.run().await,
```

Comment:
How will this extend to external triggers or is that out of scope for now?

Reply:
Out of scope for this PR. We may want to support `spin trigger <custom-type>` at some point but I don't think it is necessary immediately; `spin up` can resolve those custom types directly.